Packs/SailPointIdentityIQ/Integrations/SailPointIdentityIQ/SailPointIdentityIQ.yml

```diff
@@ -81,6 +75,14 @@ script:
       name: active
     - description: Numeric value of baseline risk score, users above this will be returned.
       name: risk
+    - auto: PREDEFINED
```

Comment:
I would suggest to do this as a free string and not predefined so users will can use this argument for other custom filters as well. (you can put in the description some examples)